chart/templates/serviceaccount.yaml

```diff
@@ -1,30 +1,39 @@
+  {{- if .Values.serviceAccount.create -}}
```

Comment:
No change in values.yaml to define/document the default for this ?
Can `unleash-service` work if `.Values.serviceAccount.create` is false or not set ?

Reply:
The variable is already defined in the `values.yaml` file. The variable is also documented inside the YAML file, but I can also add it to the README if you find it necessary.

Reply:
If this needs to be set to true the first time someone installs the unleash-service I would add that to the README.md
What concerns me is that before this PR we always defined the service account and the rolebinding while now that only happens if that value is 'true' so I want to make sure that the clean install use case still works :)

Reply:
I documented the service account creation and how you can disable it in the README.